pkgs/by-name/fr/friction-graphics/package.nix

```diff
@@ -0,0 +1,105 @@
+{ lib
+, clangStdenv
+, cmake
+, expat
+, fetchFromGitHub
+, ffmpeg_4-full
+, fontconfig
+, freetype
+, glib
+, gn
+, harfbuzzFull
+, icu
+, libGL
+, libjpeg_turbo
+, libpng
+, libsForQt5
+, libunwind
+, libwebp
+, ninja
+, pcre2
+, pkg-config
+, python3
+, qt5Full
+, xorg
+, zlib
+}:
+
+clangStdenv.mkDerivation rec {
+  pname = "friction.graphics";
+  version = "0.9.6";
+
+  src = fetchFromGitHub {
+    fetchSubmodules = true;
+    owner = "friction2d";
+    repo = "friction";
+    rev = "v${version}";
+    hash = "sha256-QjwkijsTAEEN1VqGtIfsNsHTM1DIRUp+hfABuXoj+nU=";
+  };
+
+  buildInputs = [
+    expat
+    ffmpeg_4-full
+    fontconfig
+    freetype
+    glib
+    gn
+    harfbuzzFull
+    icu
+    libGL
+    libjpeg_turbo
+    libpng
+    libsForQt5.qscintilla
+    libunwind
+    libwebp
+    ninja
```

Comment:
ninja goes to nativeBuildInputs

Reply:
sorry for the late response — done, and tested